.github/workflows/ci.yml

```diff
@@ -12,24 +15,30 @@ jobs:
           - macOS-latest
           - ubuntu-latest
           - windows-latest
-        python-version: [ '3.7','3.8','3.9' ]
+        python-version: [ '3.8','3.9', '3.10' ] # '3.7' no longer supported
```

Comment:
Some of the libraries (e.g. networkx & subdependencies) no longer support 3.7, so I've removed it and updated to 3.10 here

Reply:
We may need to keep up pace with the latest python version.

Reply:
Yeah good point, maybe 3.11 in future.